README.md

```diff
@@ -1,75 +1,40 @@
+`solid-client-authn` is a suite of libraries designed to authenticate with Solid identity servers.
 The libraries share a common API and include different modules for different deployment environments:
 
-- `solid-client-authn-browser` can be used to help build web apps in the browser.
-- `solid-client-authn-node` is **planned** to help build server-side and console-based apps.
+- `solid-client-authn-browser` can be used to build web apps in the browser.
+- `solid-client-authn-node` is **planned** to build server-side and console-based apps. (NOT AVAILABLE YET)
 
-@inrupt/solid-client-authn libraries are part of a family open source JavaScript libraries designed to support developers building Solid applications.
+@inrupt/solid-client-authn libraries are part of a suite open source JavaScript libraries designed to support developers building Solid applications.
 
 # Inrupt Solid JavaScript Client Libraries
 
 ## Data access and permissions management - solid-client
 
 [@inrupt/solid-client](https://docs.inrupt.com/client-libraries/solid-client-js/) allows developers to access data and manage permissions on data stored in Solid Pods.
 
-## Authentication - solid-client-authn
-
-[@inrupt/solid-client-authn](https://github.com/inrupt/solid-client-authn) allows developers to authenticate against a Solid server. This is necessary when the resources on your Pod are not public.
+## Authentication - solid-client-authn-browser
+[@inrupt/solid-client-authn-browser](https://github.com/inrupt/solid-client-authn-browser) allows apps running on a browser to authenticate against a Solid server. This is necessary when the resources on your Pod are not public.
 
 ## Vocabularies and interoperability - solid-common-vocab-rdf
 
 [@inrupt/solid-common-vocab-rdf](https://github.com/inrupt/solid-common-vocab-rdf) allows developers to build interoperable apps by reusing well-known vocabularies. These libraries provide vocabularies available as constants that you just have to import.
 
-# Authentication libraries
-
-The different `solid-client-authn` libraries all share the same interface ([see API reference](https://docs.inrupt.com/client-libraries/api/javascript/solid-client-authn-browser/)). `solid-client-authn` includes two APIs for building applications: a "Single Session API" and a "Multi Session API".
-
-## Usage
-
-### Single session API
-
-`solid-client-authn`'s single session API is designed to provide a streamlined experience when implementing authentication in apps that focus on just one user.
-You'd use the single session API to:
-
-- Create a single page web application that only one user logs into at a time.
-- Make a bot that only represents one user.
-- Build a mobile application that only one user logs into at a time.
+# Installation
 
-#### Examples
+For the latest stable version of solid-client-authn-browser:
 
-- [Single-session the web browser with your own bundler](examples/single/bundle)
```

Comment:
The example moved to `packages/browser/examples/single/bundle`. I think we should fix the link rather than remove it: providing runnable code snippets is quite useful.

Reply:
@nicolasmondada What do you think ?

Reply:
We should clean up the examples folder first (having non-working examples won't help), document the example on the docs, and then add it back to the readme. I can create a separate ticket for this.